### DISCUSSION TITLE
How to skip a code block?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#8146] [Q&A] [answered]
### Description

May I ask how to show a code block without execuiting it, i.e. skip the code block? 

Thanks.

**Comment by mcanouil** [comment, marked as answer]
What does "skip" means exactly?

Use code cell option `eval: false` or don't use brackets in the code definition as documented in Quarto website.

**Comment by hopezh** [comment]
Thanks, @mcanouil 

The `eval: false` option of a code block is what I'm looking for. 

For example, I'd like to show the content of a python `NamedTuple` named `aux` without triggering quarto to evaluate it as a python code block: 

Source code in the qmd file:
<img width="1248" alt="Screenshot 2024-01-07 at 16 24 19" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/6fe052c9-a402-412f-9084-8b54c1307eff">'

What is rendered as a webpage:
<img width="1378" alt="Screenshot 2024-01-07 at 16 29 16" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/e4b762a2-f21f-4bcd-9a7b-4894ee9bbebf">

**Reply by mcanouil** [reply]
Not using the brackets is way simpler and less typing if you never intend to evaluate the code.
````
```python
```
````

Here you are saying "I want the code block to be a computation code cell but I don't want it to compute anything".
While you could just "it's code".
